osd/invalid_iaas_credentials.json

```diff
@@ -3,6 +3,6 @@
     "service_name": "SREManualAction",
     "cluster_uuid": "${CLUSTER_UUID}",
     "summary": "Action required: delete inaccessible cluster",
```

Comment:
Should update this field also to remove the delete reference.

"Action required: Restore missing cloud credentials"